webapp/lib/view.dart

```diff
@@ -292,6 +292,9 @@ class PackageConfiguratorView extends BaseView {
     _buildSidebarPartial();
     packageConfiguratorViewElement.append(_packageConfiguratorSidebar);
     packageConfiguratorViewElement.append(_packageConfiguratorContent);
+    packageConfiguratorViewElement.onClick.listen((_) {
+      document.querySelectorAll('.add-tag-dropdown').forEach((dropdown) => dropdown.remove());
+    });
```

Comment:
this is a little bit buggy in that if you click on the header, the menu won't disappear. Instead of adding additional listeners to the other DOM elements outside this element, I suggest the following pattern:
```
/// this is just pseudocode - the idea is to create a one-off click listener on the document
/// by cancelling its subscription after the first time it's been fired

void _addTagDropdown(...) {
  // do stuff to create and show the dropdown (just as it is currently at line 521)
  // and then create the one-off click listener
  var documentOnClickSubscription;
  documentOnClickSubscription = document.onClick.listen((e) {
    e.stopPropagation();
    dropdownMenu.remove();
    documentOnClickSubscription.cancel();
  );
});
```
You will still need to have an `onClick` listener on the dropdown which will select the tag, and that can also cancel the `documentOnClickSubscription`.

Reply:
I see how buggy this was. This is a better approach. Made the changes.